packages/strapi-plugin-upload/config/schema.graphql.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     }
   `,
   mutation: `
-    upload(refId: ID, ref: String, field: String, source: String, file: Upload!): UploadFile!
+    upload(refId: ID, ref: String, field: String, source: String, info: FileInfoInput, file: Upload!): UploadFile!
```

Comment:
You haven't created the FIleInfoInput type in the schema :)

Reply:
Yes, because it already exists :)
https://github.com/strapi/strapi/blob/5b4e317d75b603376967803bae6cc4872da43642/packages/strapi-plugin-upload/config/schema.graphql.js#L6-L10

Reply:
I'm currently using this on my project.
It worked well for me :)

Reply:
Ho my bad I didn't remember we already had it :D 
Everything looks good to me but I would like you to add some tests in packages/strapi-plugin-upload/tests/graphqlUpload.test.e2e.js

Reply:
Is a right way to do e2e test by running `node test/e2e.js`?

Reply:
@alexandrebodin I've added the test. Please check